### ISSUE TITLE
Preferred place for donations

### ISSUE THREAD

**Issue by WorikQCI** [caldwell/build-emacs#125]
[Here you say](https://emacsformacosx.com/donate) `In November 2011 I used Funded By Me`.  That organsisation has been taken over (merged into???) some other outfit.

Is that still where you would like donations?  I am reluctant to open an account with them without you confirming they are still useful.

Have you considered Github Sponsor?

I have no opinion, I am looking for information both on how best to send you money for beer (or whatever) and also get some idea of the state of donation systems surrunding Free Software.

Thank you for your hard work!

**Comment by caldwell** [comment]
> Is that still where you would like donations? I

No, Funded By Me was a Kickstarter-like thing and the campaign is long over. The donate page you linked has a paypal "donate" button on it and a bitcoin address. Those are the only 2 payment options I have at the moment. I tweaked the wording on the page and removed the dead link to Funded By Me, so hopefully the page will be a little more clear.

I haven't considered Github Sponsor but I'll look into it.
